src/perf_tool_backend.ml

```diff
@@ -107,6 +107,7 @@ module Perf_line = struct
   let report_fields = "pid,tid,time,flags,addr,sym,symoff"
 
   let to_event line =
+    try
```

Comment:
Nit: indenting of the `try` body. https://github.com/janestreet/magic-trace/pull/30 should help.

Reply:
formatting passes now